src/services/utxoAtPoint.ts

```diff
@@ -47,12 +42,15 @@ LIMIT $4::word31type OFFSET $5::word31type;
 
 export const utxoAtPoint =
   (pool: Pool) => async (req: Request, res: Response) => {
-    if (!req.body || !req.body.addresses) {
-      throw new Error("error, no addresses.");
+    if (!req.body) {
```

Comment:
duplicate same logic from here https://github.com/Emurgo/yoroi-graphql-migration-backend/blob/223734a5fdcfb25714e6060d9a986599e30d9058/src/services/utxoDiffSincePoint.ts#L41, for the `addresses` + `body`